Content.Shared/Chemistry/Containers/Components/ContainerSolutionComponent.cs

```diff
@@ -0,0 +1,33 @@
+using Content.Shared.Chemistry.Containers.EntitySystems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Chemistry.Containers.Components;
+
+/// <summary>
+/// 
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedSolutionContainerSystem))]
+public sealed partial class ContainerSolutionComponent : Component
+{
+    [DataField]
+    public EntityUid Container;
+
+    [DataField]
+    public string Name = default!;
+}
+
+
+[Serializable, NetSerializable]
+public sealed partial class ContainerSolutionState : ComponentState
```

Comment:
auto comp states

Reply:
IIRC auto comp states can't handle the EntityUid because it's not netserializable.

Reply:
It automatically converts them to `NetEntity` and back